CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 1.0.2 (2022-03-11)
```

Comment:
Please, don't update the release version/date in the PR; we don't know when this is gonna be published 🙂

Reply:
Addressed in [b3741a5](https://github.com/palkan/active_event_store/pull/9/commits/b3741a59fd5fbf20d89b22e840969c1297440e77)